test/test_distributed.py

```diff
@@ -700,13 +734,79 @@ def test_all_gather_multigpu(self):
         self._test_all_gather_multigpu_helper(group, group_id, rank,
                                               rankToGPUMapping)
 
+    # END TO END TEST FOR DISTRIBUTEDDATAPARALLEL
+    def _test_DDP_helper(self, model, input_var, target, loss, optimizer):
+        model.train()
+        output = model(input_var)
+        l = loss(output, target)
+        l.backward()
+        optimizer.step()
+        return model
```

Comment:
It would be clearer to make this a function that has a side effect and doesn't return anything. Returning the model seems to imply that it's a new thing, while it really is what was given as an argument (and we're unnecessarily re-assigning it)